src/Logger.cpp

```diff
@@ -52,7 +53,6 @@ void cLogger::LogSimple(AString a_Message, eLogLevel a_LogLevel)
 	#endif
 
 
-	cCSLock Lock(m_CriticalSection);
```

Comment:
This change is not needed, logging is supposed to be thread-unsafe on the line-level - unrelated threads have no proper ordering.
You're now creating more load and more contention on the logging mutex, and there's no real benefit, threads' output can still be mixed line-wise in any order.